pkg/private/xtest/graphupdater/graph.go

```diff
@@ -104,6 +104,17 @@ func (g *Graph) Write(w io.Writer, descName string) (int, error) {
 	return total, nil
 }
 
+func (g *Graph) WriteIfIDs(w io.Writer) (int, error) {
+	total := 0
+	lines := []string{strings.Join(g.ifIDs(), "\n")}
+	n, err := w.Write([]byte(strings.Join(lines, "\n")))
+	total += n
```

Comment:
```suggestion
	n, err := w.Write([]byte(strings.Join(g.ifIDs(), "\n")))
	total += n
```

correct me if I'm wrong but lines always has len == 1 so there's no need to join lines.

Reply:
Good catch, thank you!